cmd/maya-apiserver/cstor-operator/cspc/handler.go

```diff
@@ -162,9 +181,15 @@ func (pc *PoolConfig) create(pendingPoolCount int, cspc *apis.CStorPoolCluster)
 	glog.V(4).Infof("Lease acquired successfully on cstorpoolcluster %s ", cspc.Name)
 	defer newSpcLease.Release()
 	for poolCount := 1; poolCount <= pendingPoolCount; poolCount++ {
+
+		message := fmt.Sprintf("Provisioning pool %d/%d ", poolCount, pendingPoolCount)
+		pc.Controller.recorder.Event(cspc, "Info", "Create", message)
```

Comment:
instead of this, how about raising event for success case.. same goes for log in next line